grails-app/domain/org/pih/warehouse/invoice/Invoice.groovy

```diff
@@ -134,7 +134,9 @@ class Invoice implements Serializable {
             dateDue: dateDue,
             datePaid: datePaid,
             currencyUom: currencyUom?.id,
-            vendor: party?.id
+            vendor: party?.id,
+            totalCount: invoiceItems.size(),
```

Comment:
@DominikaHatala I am getting NPE here. Should be `invoiceItems?.size()?:0`

Reply:
@awalkowiak That's right. Fixed.